docs/Mapping-a-Controller-for-Intellivision.md

```diff
@@ -242,7 +242,7 @@ For games that require Intellivoice support, add an additional "jzintv-voice" li
 <pre>jzintv-voice = "/opt/retropie/emulators/jzintv/bin/jzintv -p /home/pi/RetroPie/BIOS -q --kbdhackfile=/home/pi/RetroPie/roms/intellivision/hackfile.cfg %ROM%" -v1</pre>
 For games that require Intellivision ECS support, add an additional "jzintv-ecs" line that includes the '-s1' parameter to `/opt/retropie/configs/intellivision/emulators.cfg` per below:
 <pre>jzintv-ecs = "/opt/retropie/emulators/jzintv/bin/jzintv -p /home/pi/RetroPie/BIOS -q --kbdhackfile=/home/pi/RetroPie/roms/intellivision/hackfile.cfg %ROM%" -s1</pre>
-Note that games requiring Intellivoice and/or ECS support must be assigned to the lines defined above using the [Runcommand Launch Menu](https://github.com/RetroPie/RetroPie-Setup/wiki/runcommand).
+Note that games requiring Intellivoice and/or ECS support must be assigned to the lines defined above using the [Runcommand Launch Menu](runcommand).
```

Comment:
`[Runcommand Launch Menu](runcommand)` should be `[Runcommand Launch Menu](Runcommand#runcommand-launch-menu)`.